docs/sources/getting-started.md

```diff
@@ -73,6 +73,15 @@ export OPENHAB_HTTP_PORT=8011
 export OPENHAB_HTTPS_PORT=8444
 ```
 
+## Configuring additional JVM options
```

Comment:
Maybe we should directly refer to the cases, where this is needed (we are here on a "getting-started"), which is specifically defining a non-standard serial port.